.circleci/config.yml

```diff
@@ -17,7 +17,7 @@ jobs:
       - run:
           name: Setup Build
           command: |
-            mkdir -p $CIRCLE_TEST_RESULTS
+            mkdir -p ~/test-results
```

Comment:
can't all instances of `test-results` just be `$CIRCLE_TEST_REPORTS` ?

Reply:
That's what I tried before, see the commit history. Didn't work - I am still trying different things.

Reply:
Tried again and get this:

Reply:
Unfortunately I don't know enough to help at t his point :( I think it's fine using the raw string values as you did before.

Reply:
After just trying it, I investigated some more. Seems using environment variables inside the config.yml is not supported in Cirlce CI 2.0: